bon/tests/integration/ui/compile_fail/errors.stderr

```diff
@@ -210,14 +210,14 @@ note: required by a bound in `ExampleBuilder::<(__X, __Y, __Z)>::build`
    |            ------- required by a bound in this associated function
    = note: this error originates in the derive macro `Builder` (in Nightly builds, run with -Z macro-backtrace for more info)
 
-error[E0277]: can't finish building yet; the member `ExampleBuilder__renamed` was not set
+error[E0277]: can't finish building yet; the member `renamed` was not set
```

Comment:
I'm curious how this works when there are two structs with the fields that are named the same . Won't rustc prepend the module name here to avoid confusion?

Reply:
Unfortunately it does. I was testing mostly with `cargo doc` and there are no issues there. I will try to find solution and add it to compile tests.

Reply:
Yeah, that's unfortunate, and that was one of the reasons why I prepended the builder name to the field here in the first place (although I admittedly, did test my assumption 😳).

Maybe this could be worked around by using a conditional `#[cfg(doc)]`, which I already do as a hack to keep return types of setters shoter in rustdoc output:

https://github.com/elastio/bon/blob/add06f68e5571da4325227757000d2dbb0b48763/bon-macros/src/builder/builder_gen/setter_methods.rs#L190-L194